harvester_e2e_tests/integrations/test_5_vm_networks_interact.py

```diff
@@ -317,14 +300,19 @@ def test_mgmt_network_connection(
                                             " to VM on management interface"
                                             f" {mgmt_ip} from external host")
 
-        # cleanup VM
-        delete_vm(api_client, unique_name, wait_timeout)
+        # teardown
+        code, data = api_client.vms.get(vm_name)
+        vm_spec = api_client.vms.Spec.from_dict(data)
+        vm_checker.wait_deleted(vm_name)
```

Comment:
qu: For all `#teardown` new implementations -> do we want to pass in a wait_timeout for `wait_deleted` [from the virtualmachines.py fixture](https://github.com/harvester/tests/blob/586d6a718573a718f53ee3e7674fb0c647bf373f/harvester_e2e_tests/fixtures/virtualmachines.py#L224-L237) it's keyword arg of `endtime`?
Maybe something like `endtime=wait_timeout+datetime.now()` (pseudo code)